### DISCUSSION TITLE
posts/a-quick-guide-to-apples-freeform-app/

### DISCUSSION THREAD

**Discussion by giscus** [nerdymomocat/nerdymomocat.github.io#9] [Announcements]
# posts/a-quick-guide-to-apples-freeform-app/

This might be the whiteboard app that you have always wanted (or not)?

https://nerdymomocat.github.io/posts/a-quick-guide-to-apples-freeform-app/

**Comment by harshjn** [comment]
Nice. Thanks for the article. Did you write it in Notion?

**Reply by nerdymomocat** [reply]
I did! I use webtrotion to render it after.
